be/src/exec/hdfs_scanner.cpp

```diff
@@ -105,6 +105,9 @@ Status HdfsScanner::_build_scanner_context() {
         column.col_type = slot->type();
         column.slot_id = slot->id();
         column.col_name = slot->col_name();
+        column.decode_needed =
```

Comment:
Add some comment, like `only parquet reader is using this flag now`

Reply:
> Add some comment, like `only parquet reader is using this flag now`

I think orc may could do this optimization too.

Reply:
You are right, but not supported now, so afraid will confuse other people.

Reply:
Get it.